src/odemis/model/_dataio.py

```diff
@@ -20,7 +20,7 @@
 You should have received a copy of the GNU General Public License along with
 Odemis. If not, see http://www.gnu.org/licenses/.
 '''
-from abc import ABCMeta, abstractmethod
+from abc import abstractmethod
 
 
 class DataArrayShadow(object):
```

Comment:
Make ABCMeta to DataArrayShadow, and make add a `tile_shape=None` argument, which should be passed in the same conditions as maxzoom.